### ISSUE TITLE
TextFormatFlags.NoPadding not working with .Net 5 Windows App (Core).

### ISSUE THREAD

**Issue by smToDo** [dotnet/winforms#3823]
* .NET Core Version: 
.NET SDK 5.0.100-preview.8.20417.9

* Have you experienced this same bug with .NET Framework?: 
No
It is also not reproducible with .Net Core 3.1.

**Problem description:**

The TextRenderer.DrawText method ignores the TextFormatFlags.NoPadding flag and render string with padding. But the MeasureText method works correctly with this flag.
This sample measure text size and render it, when I run this code with the .Net Framework / .Net Core 3.1/ .Net 5 old builds - this code works correctly, but after update VS to Version 16.8.0 Preview 2.1, this code works incorreclty.

Sample:
```
protected override void OnPaint(PaintEventArgs e)
        {
            base.OnPaint(e);

            var g = e.Graphics;
            var text = "Text Text Text";
            var font = Font;
            var width = int.MaxValue;
            var flags = TextFormatFlags.NoPadding;
            var size = TextRenderer.MeasureText(g, text, font, new Size(width, 0), flags);
            var r = new Rectangle(Point.Empty, size);
            TextRenderer.DrawText(g, text, font, r, ForeColor, flags);
            System.Diagnostics.Debug.WriteLine(r);
        }
```
Result:
.Net Framework 4.5.2:
![NetF452](https://user-images.githubusercontent.com/20538865/91958014-75827e00-ed0f-11ea-895e-5a79fb676821.png)
.Net 5:
![Net5](https://user-images.githubusercontent.com/20538865/91958036-7f0be600-ed0f-11ea-9bc9-d24be9a264f3.png)

**Expected behavior:**
The TextFormatFlags.NoPadding - Does not add padding to the bounding rectangle.

**Minimal repro:**
1. Run attached app:
[DrawText_issue.zip](https://github.com/dotnet/winforms/files/5161126/DrawText_issue.zip)
2. Observe - NoPadding doesn't work with the DrawText method.

**Comment by IrinaPykhova** [comment]
Please note that we consider this issue as showstopper. For us it breaks behavior both for pure .Net 5 applications and for .Net 4.5.2 libraries used in .Net 5 apps. Text wraps unexpectedly in wrong places

**Comment by JeremyKuhne** [comment]
Investigating...

**Comment by JeremyKuhne** [comment]
I know what the problem is, working on the fix.

**Comment by JeremyKuhne** [comment]
The fix is in for RC2 (#3873).

**Comment by Ashley-Li** [comment]
Verified this issue with .Net SDK 5.0.100-rc.2.20460.32, this issue is fixed. After run attached app(DrawText_issue.zip),the TextFormatFlags.NoPadding - does not add padding to the bounding rectangle,TextFormatFlags.NoPadding is working with .Net 5 Windows App (Core). Please see below result:
![image](https://user-images.githubusercontent.com/56664234/92869639-a06a8180-f435-11ea-93f5-40cf9bb25ec5.png)

**Comment by smToDo** [comment]
Hi guis,
I checked Preview 3, now MeasureText ignored NoPadding too.

**Comment by RussKie** [comment]
@smToDo as @Ashley-Li indicated the issue is fixed in SDK 5.0.100-rc.**2.20460.32**
